src/averageAges.js

```diff
@@ -14,14 +14,23 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+const MALE = 'f';
+const FEMALE = 'f';
+const ONE_CENTURY = 1000;
```

Comment:
The value of `ONE_CENTURY` is set to 1000, which is incorrect as a century consists of 100 years, not 1000. It should be set to 100.